### ISSUE TITLE
Queries return all objects in class

### ISSUE THREAD

**Issue by pantaryl** [ParsePlatform/parse-server-example#7]
Hello there –

I believe I've set this up properly...

When I go to perform a query on my database, I search { "where : { "showId" : #### } }.

No matter what my ##### is, I always receive the first thing in the database.

Any ideas?

**Comment by pantaryl** [comment]
Actually, it seems that I receive all entries for that class, regardless of the query.

**Comment by gfosco** [comment]
Please provide the code you are using.

**Comment by pantaryl** [comment]
I'm using the example .js file for the server itself with my keys placed in.

I'm just trying a CURL at the moment:

```
curl -X GET \
  -H "X-Parse-Application-Id: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX" \
  -H "X-Parse-Master-Key: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX" \
  -G \
  --data-urlencode 'where={"showId":"35540"}, limit=1' \
  http://127.0.0.1:1337/parse/classes/TvShow
```

Without fail, the results to this query will always be the current contents of the TvShow class. Currently there are only two objects in there as I try to figure this out.

I believe I've set up the indexes in Mongo properly. I added an index for the showId field as text.

This is the return:
`
{"results":[{"objectId":"JD2F4egyt3","showId":"35540"},{"objectId":"xjlkafUShl", "showId":"35541"}]}
`
